tests/particularUserRoute.test.js

```diff
@@ -0,0 +1,16 @@
+const request = require('supertest')
```

Comment:
this test should be in tests/routes/ and then have the same name as your route file.